dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebValidation.java

```diff
@@ -18,7 +18,8 @@
     public static ValidationPattern DICOMWEB_STUDIES_VALIDATION =
```

Comment:
Let's delete the DICOMWEB_STUDIES_VALIDATION,  STUDIES, and STUDIES_PATH variables since they aren't used any more, as well as the associated code

Reply:
done